libraries/AP_Scripting/lua_bindings.cpp

```diff
@@ -37,6 +39,129 @@ int lua_micros(lua_State *L) {
     return 1;
 }
 
+int lua_mavlink_init(lua_State *L) {
+    binding_argcheck(L, 2);
+    WITH_SEMAPHORE(AP::scripting()->mavlink_data.sem);
+    // get the depth of receive queue
+    const int queue_size = luaL_checkinteger(L, -1);
+    // get number of msgs to accept
+    const int num_msgs = luaL_checkinteger(L, -2);
```

Comment:
```suggestion
    const int queue_size = get_uint32(L, -1, 0, 25);
    // get number of msgs to accept
    const int num_msgs = get_uint32(L, -2, 0, 25);
```
We have helpers now, you also have set some reasonable limits.

Reply:
fixed